packages/sdk/src/conversation/middlewares/commandMiddleware.ts

```diff
@@ -62,36 +62,39 @@ export class CommandResponseMiddleware implements Middleware {
       ssoHandler.triggerPatterns
     );
     this.ssoCommandHandlers.push(ssoHandler);
-    this.commandHandlers.push(ssoHandler);
     this.hasSsoCommand = true;
   }
 
   public async onTurn(context: TurnContext, next: () => Promise<void>): Promise<void> {
     if (context.activity.type === ActivityTypes.Message) {
       // Invoke corresponding command handler for the command response
       const commandText = this.getActivityText(context.activity);
-
+      let alreadyProcessed = false;
```

Comment:
Just realized one thing: there may be cases that a command text can match both a commandHandler and a ssoCommandHandler.
What's our attitude towards this scenario? An easier way is that we assume there shouldn't be intersection and document this for users. But I'm not sure whether we can make this assumption.

Reply:
From original design command bot will stop processing handlers when the first command handler is matched. So we can give some comments to tell user about this

Reply:
@MuyangAmigo how about your suggestions?

Reply:
For now we can take the assumption that developers want to build SSO/NON-SSO commands separately just not to over design at the beginning. We can always iterate this over when there's customer signal or specific asks come along.

Reply:
Thanks, resolving this comment.